src/Components/3DV/SceneViewWrapper.tsx

```diff
@@ -183,6 +185,7 @@ const SceneViewWrapper: React.FC<ISceneViewWrapperProps> = (props) => {
                 objectStyle={sceneThemeState.objectStyle}
                 onCameraMove={addInProps?.onCameraMove ? cameraMove : undefined}
                 {...svp}
+                outlinedMeshitems={sceneViewState.outlinedMeshItems}
```

Comment:
It would be good to add a story that illustrates how to use the properties that are set by the context.  In fact add the context to the existing stories that contains no properties just so we can add the context based properties to it as we remove them from props as implicit documentation of how the context works for consumers of SceneViewWrapper